tests/rpm-ostree/main.yml

```diff
@@ -0,0 +1,571 @@
+---
+# vim: set ft=ansible:
+#
+#   This is a general test suite for rpm-ostree that exercises several of the
+# rpm-ostree subcommands.  These tests assume the atomic host is provisioned
+# with the latest version.  These tests are meant to be run in succession.
+#
+- name: rpm-ostree - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    # This playbook requires Ansible 2.2 and an Atomic Host
+    - role: ansible_version_check
+      avc_major: "2"
+      avc_minor: "2"
+      tags:
+        - ansible_version_check
+
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+#####################################################################################
+#
+# rpm-ostree deploy version
+#   - validates deploying by version string
+#   - validates cleanup of rollback deployment
+#
+#####################################################################################
+
+- name: rpm-ostree - deploy by version
+  hosts: all
+  become: yes
+
+  tags:
+    - deploy_version
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - name: Get current refspec
+      shell: cat $(ostree admin --print-current-dir).origin | grep refspec | cut -d= -f2
+      register: refspec
+
+    - name: Get json ouput from rpm-ostree status
+      command: rpm-ostree status --json
+      register: ros_status
+
+    - name: Convert to json
+      set_fact:
+         ross_json: "{{ ros_status.stdout | from_json }}"
+
+    - name: Set current version
+      set_fact:
+         head_version: "{{ ross_json['deployments'][0]['version'] if ross_json['deployments'][0]['booted'] else ross_json['deployments'][1]['version'] }}"
+
+    - name: Pull last two commits
+      command: ostree pull --subpath /usr/share/rpm --depth=1 {{ refspec.stdout }}
```

Comment:
Ah, we started doing this in our deployment playbooks/ jobs:

```
commit 7857fe3
Author: Micah Abbott <miabbott@redhat.com>
Date:   Wed Oct 12 11:01:25 2016 -0400

    ans_atomic_info: fetch commit and file/RPM data
    
    This changes the playbook to pull the file/RPM data for use in determining
    the the package differences between HEAD^ and HEAD of the stream being
    tested.
    
    We had avoided doing this in the past because an `ostree pull
    --commit-metatdata-only` of the CAHC stream would fetch the deltas
    being generated between the two commits.  Thus, the `rpm-ostree db
    diff` would return the desired results.
...
```

I think in this instance we are safe just pulling the commit metadata